src/network/networkprotocol.h

```diff
@@ -589,27 +599,31 @@ enum ToClientCommand
 		u32 id
 	*/
 
-	TOCLIENT_NUM_MSG_TYPES = 0x54,
+	TOCLIENT_SRP_BYTES_S_B = 0x60,
+	/*
+		Belonging to AUTH_MECHANISM_LEGACY_PASSWORD and AUTH_MECHANISM_SRP.
+
+		u16 command
+		std::string bytes_s
+		std::string bytes_B
+	*/
+
+	TOCLIENT_NUM_MSG_TYPES = 0x61,
 };
 
 enum ToServerCommand
 {
-	TOSERVER_INIT = 0x02,
+	TOSERVER_INIT = 0x0f,
```

Comment:
In fact, if you could add SRP and this new _INIT packet together at 
the lower opcodes it could be good. Legacy upper is not a problem, in 1
year and more we could remove it :)